kotlin-detect-config.yaml

```diff
@@ -171,26 +181,33 @@ style:
     SerialVersionUIDInSerializableClass:
         excludes:
             - "**/examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/clusterinteraction/HistoryCommand.kt"
+            - "**/src/controller/java/generated/java/**/*"
     UnusedPrivateMember:
         excludes:
             - "**/examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/setuppayloadscanner/BarcodeFragment.kt"
+            - "**/src/controller/java/generated/java/**/*"
             - "**/src/controller/java/src/chip/onboardingpayload/OnboardingPayload.kt"
     UseCheckOrError:
         excludes:
+            - "**/src/controller/java/generated/java/**/*"
             - "**/src/controller/java/src/chip/tlv/tags.kt"
             - "**/src/controller/java/src/chip/tlv/types.kt"
     UseRequire:
         excludes:
+            - "**/src/controller/java/generated/java/**/*"
             - "**/src/controller/java/src/chip/jsontlv/JsonToTlv.kt"
             - "**/src/controller/java/src/chip/onboardingpayload/Verhoeff10.kt"
     UtilityClassWithPublicConstructor:
         excludes:
+            - "**/src/controller/java/generated/java/**/*"
```

Comment:
This also seems applicable to generated code.

Reply:
I removed.